src/main/java/tracko/model/item/Price.java

```diff
@@ -30,23 +30,31 @@ public Price(BigDecimal price) {
      * @param test The price to validate.
      * @boolean Representing if the input is a valid price.
```

Comment:
The JavaDoc here should be
`@return True if input is a valid price`
instead of
`@boolean ....`

Reply:
Updated!